doc/frontend/frontend.md

```diff
@@ -0,0 +1,24 @@
+# Frontend documentation
+
+## vuejs components :
+
+- activity-navbar :
+  > display description room and if a user wrinting in the input
+
+- messages :
+  >display messages 
+
+- room-list :
+  > display all created rooms
+   
+   --> room-entry :
+    > when the user click on the room, he can change her name, description or delete this room
```

Comment:
because there isn't any concept of room ownership, changing the name or description of a room won't be possible (for now). here, deleting a room would probably be quitting it, and we might consider a room "dead" once no one is inside anymore. Clicking on the room would select it and let the user talk there.

Reply:
Not implemented in the front.